ArduPlane/quadplane.cpp

```diff
@@ -4112,22 +4112,13 @@ float QuadPlane::get_land_airspeed(void)
 {
     const auto qstate = poscontrol.get_state();
     if (qstate == QPOS_APPROACH ||
-        plane.control_mode == &plane.mode_rtl) {
+        plane.control_mode == &plane.mode_qrtl) {
         const float cruise_speed = plane.aparm.airspeed_cruise_cm * 0.01;
-        float approach_speed = cruise_speed;
-        float tecs_land_airspeed = plane.TECS_controller.get_land_airspeed();
-        if (is_positive(tecs_land_airspeed)) {
-            approach_speed = tecs_land_airspeed;
-        } else {
-            if (qstate == QPOS_APPROACH) {
-                // default to half way between min airspeed and cruise
-                // airspeed when on the approach
-                approach_speed = 0.5*(cruise_speed+plane.aparm.airspeed_min);
-            } else {
-                // otherwise cruise
+        float approach_speed = plane.TECS_controller.get_land_airspeed();
```

Comment:
Currently we always use land airspeed if set, now we only use it if not in `QPOS_APPROACH`. Now in `QPOS_APPROACH` we always fly at cruise speed.

Reply:
the only time this function is called(aside for an arming check) in QPlane is in a Spiral approach, POS2 for QRTL , and in AUTO VTOL landing approach. 
It only returns cruise for a landing speed if -1 for Spiral 
for QPOS_APPROACH its either the entered landing speed or the 
halfway calculation....
for QRTL POS2 and any other phase of an auto vtol landing, it returns a calculated velocity.....

the new code never returns cruise speed, so Spiral approach is a change of behavior to a lower speed
so there is no change in behavior for  QPOS_APPROACH....call when in POS2 QRTL will return the landing speed instead of the calculated velocity....so maybe that should actually not be changed from RTL to QRTL but rather dropped so that the calculated velocity is used?